tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -377,6 +381,7 @@ public static int setDateOrder(int dateOrder) {
       // ResultSet ---------------------------------------------------------
     $RESULT_SET_FETCH_ROWNUM("tajo.resultset.fetch.rownum", 200),
     $RESULT_SET_BLOCK_WAIT("tajo.resultset.block.wait", true),
+    $RESULT_SET_COMPRESSION("tajo.resultset.compression", false),
```

Comment:
It also should be ``$COMPRESSED_RESULT_TRANSFER`` for consistency of ``COMPRESSED_RESULT_TRANSFER`` in SessionVars.